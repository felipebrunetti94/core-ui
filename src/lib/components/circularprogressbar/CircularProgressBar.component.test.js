import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/circularprogressbar/__snapshots__"
});
