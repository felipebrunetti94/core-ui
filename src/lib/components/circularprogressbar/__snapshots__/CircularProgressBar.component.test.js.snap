// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots CircularProgressBar Default 1`] = `
<div
  className="sc-ifAKCX krxbjr"
>
  <h3
    className="sc-EHOje ccjuFF"
  >
    With Custom Color
  </h3>
  <div
    className="sc-circularprogressbar sc-bdVaJa cRSqbK"
    color="#00ff43"
  >
    <div
      className="sc-bxivhb bUZGRr"
    >
      <h2>
        Total Capacity
      </h2>
    </div>
    <div
      className="sc-bwzfXH bWjxaj"
    >
      <svg>
        <circle
          cx="70"
          cy="70"
          r="70"
        />
        <circle
          cx="70"
          cy="70"
          r="70"
        />
      </svg>
      <div
        className="sc-htpNat ktbemA"
      >
        <h2>
          90
          %
        </h2>
        <h3>
          used
        </h3>
      </div>
    </div>
  </div>
  <h3
    className="sc-EHOje ccjuFF"
  >
    With Default Color
  </h3>
  <div
    className="sc-circularprogressbar sc-bdVaJa dNQqff"
  >
    <div
      className="sc-bxivhb bUZGRr"
    >
      <h2>
        Total Capacity
      </h2>
    </div>
    <div
      className="sc-bwzfXH bWjxaj"
    >
      <svg>
        <circle
          cx="70"
          cy="70"
          r="70"
        />
        <circle
          cx="70"
          cy="70"
          r="70"
        />
      </svg>
      <div
        className="sc-htpNat ktbemA"
      >
        <h2>
          60
          %
        </h2>
        <h3>
          used
        </h3>
      </div>
    </div>
  </div>
  <div
    className="sc-circularprogressbar sc-bdVaJa iIbTZu"
  >
    <div
      className="sc-bxivhb bUZGRr"
    >
      <h2>
        Total Capacity
      </h2>
    </div>
    <div
      className="sc-bwzfXH bWjxaj"
    >
      <svg>
        <circle
          cx="70"
          cy="70"
          r="70"
        />
        <circle
          cx="70"
          cy="70"
          r="70"
        />
      </svg>
      <div
        className="sc-htpNat ktbemA"
      >
        <h2>
          30
          %
        </h2>
        <h3>
          used
        </h3>
      </div>
    </div>
  </div>
</div>
`;
